columnphysics/icepack_itd.F90

```diff
@@ -27,15 +27,15 @@ module icepack_itd
 
       use icepack_kinds
       use icepack_parameters, only: c0, c1, c2, c3, c15, c25, c100, p1, p01, p001, p5, puny
-      use icepack_parameters, only: Lfresh, rhos, ice_ref_salinity, hs_min, cp_ice, Tocnfrz, rhoi
```

Comment:
Didn't we fix something like this in Icepack already?

Reply:
No.  The Tf/Tocnfrz was implemented on the E3SM side, and we have not moved any of those changes over to the Consortium.  In hindsight, it probably should have been done on the Consortium side and moved to the E3SM side.  I think that was early days of the E3SM Icepack version, and we probably hadn't realized that we should do changes, when possible, in the Consortium.  That approach was adopted later.